packages/office-ui-fabric-react/src/components/Button/BaseButton.tsx

```diff
@@ -386,12 +387,13 @@ export class BaseButton extends BaseComponent<IBaseButtonProps, IBaseButtonState
     }
 
     return (
-      <IconButton
+      <BaseButton
         tabIndex={ -1 }
         styles={ {
-          root: disabled ? styles.splitButtonMenuButtonDisabled : styles.splitButtonMenuButton,
-          rootChecked: styles.splitButtonMenuButtonChecked,
-          icon: disabled ? styles.splitButtonMenuIconDisabled : styles.splitButtonMenuIcon
+          root: disabled ? mergeStyles(styles.splitButtonMenuButton, styles.splitButtonMenuButtonDisabled) : styles.splitButtonMenuButton,
```

Comment:
all of the logic I believe moved to the classnames helper, so that we memoize the result rather than recompute classes every render.